ramls/anonymize-storage-loans-response.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Response schema for anonymize loans request",
+  "type": "object",
+  "properties": {
+    "anonimizedLoans": {
+      "description": "Successfully anonymized loan ids",
+      "type": "array",
+      "items":{
+        "type": "string"
+      }
+    },
+    "notAnonimizedLoans": {
```

Comment:
typo? Should it be "notAnonymizedLoans"?

Reply:
Thank you for noticing. Fixed.